StackTraceParser.cs

```diff
@@ -1,4 +1,4 @@
-#region Copyright (c) 2011 Atif Aziz. All rights reserved.
+#region Copyright (c) 2020 Atif Aziz. All rights reserved.
```

Comment:
Revert since the copyright notice should indicate the first publication year of the work:

```suggestion
#region Copyright (c) 2011 Atif Aziz. All rights reserved.
```